ViewControllers/DetailViewController.swift

```diff
@@ -203,29 +238,62 @@ class DetailViewController: UIViewController {
             movieDescriptionLabel.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor),
             movieDescriptionLabel.topAnchor.constraint(equalTo: durationStack.bottomAnchor, constant: 16),
 
-            languageStack.leftAnchor.constraint(equalTo: releaseStack.leftAnchor),
-            languageStack.topAnchor.constraint(equalTo: movieDescriptionLabel.bottomAnchor, constant: 16),
-
-            genreStack.leftAnchor.constraint(equalTo: releaseStack.leftAnchor),
-            genreStack.topAnchor.constraint(equalTo: languageStack.bottomAnchor, constant: 8)
+            collectionView.leftAnchor.constraint(equalTo: releaseStack.leftAnchor),
+            collectionView.topAnchor.constraint(equalTo: movieDescriptionLabel.bottomAnchor, constant: 16),
+            collectionView.rightAnchor.constraint(equalTo: movieDescriptionLabel.rightAnchor),
+            collectionView.heightAnchor.constraint(equalToConstant: 200),
         ])
     }
     
-    func configureWithData() {
+    private func configureWithData() {
         guard let movieDetails = movieDetails else { return }
 
         titleLabel.text = movieDetails.title
         
-        let path = movieDetails.posterPath
-            if let url = movieDataService.getMoviePosterURL(posterPath: path) {
-                imageView.kf.setImage(with: url)
-            }
+        if let path = movieDetails.posterPath {
+            imageView.kf.setImage(with: path)
+        }
         
-        releaseDateLabel.text = movieDetails.releaseDate
-        durationLabel.text = "\(movieDetails.runtime)m"
+        releaseDateLabel.text = "\(movieDetails.releaseDate)"
```

Comment:
`movieDetails.releaseDate` is a string now so we can:
```suggestion
        releaseDateLabel.text = movieDetails.releaseDate
```